themes/tiddlywiki/tight-heavier/metrics.multids

```diff
@@ -0,0 +1,8 @@
+title: $:/themes/tiddlywiki/tight-heavier/metrics/
```

Comment:
What is the reason for adding these metrics? The settings from the vanilla theme should be present.